.github/workflows/ci.yaml

```diff
@@ -125,4 +125,4 @@ jobs:
       - name: Run pytest
         shell: bash
         run: |
-          pytest --numprocesses auto -rsx --doctest-modules nltk/test
+          pytest --numprocesses auto -rsx --doctest-modules nltk
```

Comment:
`--doctest-modules` was already always on, but it didn't do much, as it only looked in `nltk/test`. Looking in `nltk` directly was broken until #3033, but it works now!

Reply:
Thanks @tomaarsen, this should make CI much more robust. I did not expect that it would be ready so soon!

Reply:
Oh!